src/testing/tests.rs

```diff
@@ -885,6 +885,139 @@ pub mod generic_wallet_tests {
         Ok(())
     }
 
+    #[async_std::test]
+    pub async fn test_wallet_freeze_unregistered<'a, T: SystemUnderTest<'a>>() -> std::io::Result<()>
```

Comment:
Can you just move this test outside of `mod generic_wallet_tests` and remove the `<'a, T>` generics? No point in instantiating this test for CAPE and Espresso if it uses the same mock system every time

Reply:
Oh yes, I'll do it.